contributed_definitions/nyaml/NXsingle_crystal.yaml

```diff
@@ -0,0 +1,31 @@
+category: base
+doc: |
+  Description of a single crystal material or a single crystalline phase in a material.
+type: group
+NXsingle_crystal(NXobject):
+  sample_orientation(NX_FLOAT):
+    unit: NX_ANGLE
+    doc: |
+      This will follow the Busing-Levy convention:
+      W. R. Busing and H. A. Levy (1967). Acta Cryst. 22, 457-464
+    dimensions:
+      rank: 1
+      dim: [[1, 3]]
+  orientation_matrix(NX_FLOAT):

```

Comment:
This originates from historic diffraction notation for single crystals, and I strongly suggest that we keep compatibility with that. Otherwise, this won't be liked by NIAC, I suppose...

Reply:
We still have the non-deprecated fields in NXsample, but they are very unorganized, so I would like to have a PR containing my suggestions here together with Markus's in PR51.